risc0/zkvm/src/host/server/exec/executor.rs

```diff
@@ -771,3 +776,58 @@ impl<'a> ExecutorImpl<'a> {
         ))
     }
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+struct ExecutorStats {
+    execution_time: Duration,
+    segment_count: usize,
+    total_prover_cycles: u64,
+    user_instruction_cycles: u64,
+    cycles_used_ratio: f64,
+}
+
+impl ExecutorStats {
+    fn new(session: &Session, elapsed: Duration) -> Result<Self> {
+        let (total_prover_cycles, user_instruction_cycles) = session.get_cycles()?;
+        let cycles_used_ratio = user_instruction_cycles as f64 / total_prover_cycles as f64 * 100.0;
+        let segment_count = session.resolve()?.len();
+
+        Ok(Self {
+            total_prover_cycles,
+            user_instruction_cycles,
+            cycles_used_ratio,
+            segment_count,
+            execution_time: elapsed,
+        })
+    }
+
+    #[instrument(name = "executor", skip(self))]
```

Comment:
What is this for?

Reply:
This simplifies log filtering, enabling us to get only the executor statistics and nothing more using simply `RUST_LOG=[executor]=<log-level>`. If we don't specify this, we'd get a log of other `info` lines in the trace as well. Filtering would require us to specify the path to the executor, which would be way longer. 

I'm not sure if there would be a better way to perform filtering though....